functions/daemon.go

```diff
@@ -198,6 +211,12 @@ func messageQueue(ctx context.Context, wg *sync.WaitGroup, server *config.Server
 	err := setupMQTT(server)
 	if err != nil {
 		slog.Error("unable to connect to broker", "server", server.Broker, "error", err)
+
```

Comment:
also, it will be safe to check on a successful connection to mq to stop the fallback thread if it's running already